cmd/multiFaAcceleration/multiFaAcceleration.go

```diff
@@ -150,14 +114,27 @@ func multiFaAcceleration(s Settings) {
 	velBed := fileio.EasyCreate(s.VelOut)
 	accelBed := fileio.EasyCreate(s.AccelOut)
 	initialVelBed := fileio.EasyCreate(s.InitialVelOut)
+	var b1OutBed, b3OutBed *fileio.EasyWriter
```

Comment:
Maybe it makes sense to re-name these since I think b1 and b3 is mostly cleaned from the code?